packages/client/index.js

```diff
@@ -134,60 +144,71 @@ function capitalize (str) {
 }
 
 // TODO: For some unknown reason c8 is not picking up the coverage for this function
-async function graphql (url, log, headers, query, variables) {
-  const res = await request(url, {
-    method: 'POST',
-    headers: {
-      ...headers,
-      'content-type': 'application/json; charset=utf-8'
-    },
-    body: JSON.stringify({
-      query,
-      variables
+async function graphql (url, log, headers, query, variables, openTelemetry) {
+  const { span, headers: telemetryHeaders } = openTelemetry ? openTelemetry.startSpanClient(url.toString(), { method: 'POST' }) : { span: null, headers: {} }
```

Comment:
I wonder if this propagates the headers correctly because the context is empty.
I think you need to get the OpenTelemetry context in https://github.com/platformatic/platformatic/blob/234330bc96f69d24ad37f683a3132302fdfb3722/packages/client/index.js#L229-L235, and then pass it down.

Reply:
You are right, we need in client (and in composer) to actually recraate the context from the headers, otherwise in the case of:

Reply:
Fixed (and added a test: https://github.com/platformatic/platformatic/blob/opentel-integration/packages/client/test/telemetry.test.js#L78)